python/pydantic_core/__init__.py

```diff
@@ -73,3 +73,19 @@ class InitErrorDetails(_TypedDict):
     loc: _NotRequired['tuple[int | str, ...]']
     input: _Any
     ctx: _NotRequired['dict[str, str | int | float]']
+
+
+class ErrorTypeInfo(_TypedDict):
+    type: ErrorType
+    message_template_python: str
+    example_message_python: str
+    message_template_json: _NotRequired[str]
+    example_message_json: _NotRequired[str]
+    example_context: 'dict[str, str | int | float] | None'
```

Comment:
Any reason to not add `from __future__ import annotations`?

Reply:
None, let's do that.